RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* Promoting `unbatch` from experimental to core API.
+* Adds `tf.enable_control_flow_v2()` and `tf.disable_control_flow_v2()`.
+* EagerTensor now support buffer interface for tensors.
+* This change bumps the version number of the FullyConnected Op to 5.
+* tensorflow : crash when pointer become nullptr.
```

Comment:
What is this from? It's useless in its current form. Remove or find the CL.

Reply:
Done!